packages/dds/tree/src/feature-libraries/editable-tree/utilities.ts

```diff
@@ -65,6 +65,11 @@ export function assertPrimitiveValueType(nodeValue: Value, schema: TreeSchema):
     }
 }
 
+/**
+ * Returns the key and the schema of the primary field out of the given tree schema.
```

Comment:
There is a ts-doc tag for return documentation ( https://tsdoc.org/pages/tags/returns/ ) which I think is preferred here to provide slightly better metadata in the documentation:
```suggestion
 * @returns the key and the schema of the primary field out of the given tree schema.
```